source/change-streams/tests/unified/change-streams.yml

```diff
@@ -854,14 +854,20 @@ tests:
         object: *globalCollection0
         arguments:
           document: { x: 1 }
+          writeConcern: 
```

Comment:
Can we leave a comment above this `writeConcern` to explain that this is needed to make sure the insert propagates and a change stream event is present on all oplogs?

Reply:
Yup, will do!